OrbitVulkanLayer/resources/VkLayer_Orbit_implicit.json

```diff
@@ -0,0 +1,18 @@
+{ 
```

Comment:
Is it clear that we want the Orbit layer to be an implicit layer? Obviously that can be changed, but we should have an idea (ideally written down in our design document) what the deployment mechanism etc. will (likely) look like.

Reply:
Yes, explicit layers need to have code on the game side, to enable them. That is nothing, that is on our radar for now. Also note, that it is totally fine for a layer to be implicit and explicit, by providing two json files.
Further, for now, that implicit layer is enabled by default,  but this is something that we can change in the future, when it comes to actual deploying. Short-term it is much more convenient to have it like this.

Reply:
Should there be a TODO to add `enable_environment` one the time comes?

Reply:
I am not sure. This is nothing that we want for sure. It depends on the way we will deploy the layer. So it will be a reasonable step as soon as we come to that point.

Reply:
Ok, makes sense.